pyproject.toml

```diff
@@ -7,14 +7,17 @@ requires-python = "~=3.10"
 dependencies = [
     "coloredlogs>=15.0.1",
     "heudiconv==1.3.3",
-    "pydicom==3.0.1",        
+    "pydicom==3.0.1",
     "pytest-mock>=3.8.2",
     "six>=1.16.0",
     "typer>=0.15.2",
-    "isort>=6.0.1",         
+    "isort>=6.0.1",
     "mne-bids==0.16.0",
     "pandas>=2.2.2",
-    "spec2nii>=0.8.0"         
+    "spec2nii>=0.8.0",
+    "aiohttp>=3.12.15",
+    "numpy==1.26.4",
+    "setuptools>=68.0.0"  
```

Comment:
Same with setuptools. Do you need it as a build-dependency? Maybe take a look at: https://learn.scientific-python.org/development/guides/packaging-simple/ for including this as a build dependency, if it's needed. I don't think you're importing it in bids_utils.py.